Source/Lib/ASM_AVX2/EbNoiseExtractAVX2.c

```diff
@@ -8,11 +8,11 @@
 #include "immintrin.h"
 #include "EbUtility.h"
 
-EB_EXTERN EB_ALIGN(16) const uint8_t filter_type[] = {
+static EB_EXTERN EB_ALIGN(16) const uint8_t filter_type[] = {
```

Comment:
filter_type is only used in this file, EB_TXTERN can be removed. Same of weak_chroma_filter

Reply:
going to rebase first to make sure mark's changes doesn't add any new conflicts, then I will remove EB_EXTERN from these